insights/parsers/grubby.py

```diff
@@ -0,0 +1,25 @@
+"""
+grubby - command ``/usr/sbin/grubby``
+=====================================
+
+This parser returns the output of the ``grubby --default-index`` command.
+
+Examples:
+
+    >>> default_index
+    0
+"""
+
+from .. import parser
+from insights.specs import Specs
+
+
+@parser(Specs.grubby_default_index)
+def grub2_default_index(context):
```

Comment:
I do suggest you wrap this parser as a Class Parser and raise a SkipException when below condition@line-24 is not satisfied.